NavigationReactNative/src/android/src/main/java/com/navigation/reactnative/TabNavigationView.java

```diff
@@ -63,6 +64,7 @@ protected void onAttachedToWindow() {
             setSelectedItemId(tabBar.selectedTab);
             autoSelected = false;
             tabBar.populateTabs();
+            this.setLabelVisibilityMode(labelVisibility);
```

Comment:
This works for the first time the tab bar renders, but what about if the user changes the `labelVisibility` later on? You can test it like this,
```jsx
const [labelVisibility, setLabelVisibility] = useState('selected');
useEffect(() => {
  setTimeout(() => {
    setLabelVisibility('hide')
  }, 1000)
), []);
<TabBar labelVisibility={labelVisibility} />
```

Reply:
Good point, hadn't considered that use case